hack/lib/strimzi.bash

```diff
@@ -232,6 +232,8 @@ EOF
 
   logger.info "Deleting existing Kafka user secrets"
 
+  oc create namespace "${EVENTING_NAMESPACE}" --dry-run=client -o yaml | oc apply -f
```

Comment:
why here? and not else, e.g. were we create other resources in this script/file ?